commands/load_balancers.go

```diff
@@ -92,6 +92,14 @@ With the load-balancer command, you can list, create, or delete load balancers,
 		"A comma-separated list of ALLOW rules for the load balancer, e.g.: `ip:1.2.3.4,cidr:1.2.0.0/16`")
 	AddStringSliceFlag(cmdLoadBalancerCreate, doctl.ArgDenyList, "", []string{},
 		"A comma-separated list of DENY rules for the load balancer, e.g.: `ip:1.2.3.4,cidr:1.2.0.0/16`")
+	AddStringFlag(cmdLoadBalancerCreate, doctl.ArgLoadBalancerDomains, "", "",
+		"A comma-separated list of domains required to ingress traffic to a global load balancer")
```

Comment:
I don't believe GLBs are available to most customers at the moment. Is that correct? If that's the case we may want to consider hiding these flags from the help output and published documentation until it is more widely available. 

Adding the `hiddenFlag()` option would do that:

```suggestion
	AddStringFlag(cmdLoadBalancerCreate, doctl.ArgLoadBalancerDomains, "", "",
		"A comma-separated list of domains required to ingress traffic to a global load balancer", hiddenFlag())
```

Reply:
👋 We're planning a limited/beta release soon, hence it may be prudent to keep these options visible? Wdyt?